manifests/base/monitoring/recording-rules.yaml

```
@@ -0,0 +1,123 @@
+          or
+          sum by (framework) (training_operator_jobs_created_total),
+        0)
+      record: training_operator:jobs_active_by_framework:sum
+      labels:
+        component: "training-operator"
+        
+    # Rule 3: Training job success rate
+    # Used for: Service health monitoring and SLA tracking
+    - expr: |
+        (
+          sum(training_operator_jobs_successful_total) / 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)